apollo/lib/checkWinCondition.tsx

```diff
@@ -118,6 +119,13 @@ function checkWinCondition(
     !isDefault && matchesPlayerList(condition.players, player);
   const ignoreIfOptional = !isDefault && condition.optional;
 
+  if (
+    !isDefault &&
+    (condition.completed?.has(player) || condition.failed?.has(player))
+  ) {
+    return false;
+  }
```

Comment:
Oh yeah, good point checking here for whether a condition has been completed before!